eng/testing/xunit/xunit.console.targets

```diff
@@ -1,87 +0,0 @@
-    <RunScriptCommand Condition="'$(XUnitClassName)' != ''">$(RunScriptCommand) -class $(XUnitClassName)</RunScriptCommand>
-
-    <!-- Add to run argument string -->
-    <RunScriptCommand>$(RunScriptCommand)$(_withCategories.Replace(';', ' -trait category='))</RunScriptCommand>
-    <RunScriptCommand>$(RunScriptCommand)$(_withoutCategories.Replace(';', ' -notrait category='))</RunScriptCommand>
-
-    <!-- User passed in options. -->
-    <RunScriptCommand Condition="'$(XUnitOptions)' != ''">$(RunScriptCommand) $(XUnitOptions)</RunScriptCommand>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.XUnitConsoleRunner"
-                      Version="$(MicrosoftDotNetXUnitConsoleRunnerVersion)"
-                      Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
-    <PackageReference Include="xunit.runner.console"
-                      Version="$(XUnitVersion)"
-                      GeneratePathProperty="true"
-                      Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
-  </ItemGroup>
-
-  <!-- Overwrite the runner config file with the app local one. -->
-  <Target Name="OverwriteDesktopTestRunnerConfigs"
-          Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and 
-                     '$(GenerateAppConfigurationFile)' == 'true' and
-                     '@(AppConfigWithTargetPath)' != ''"
-          AfterTargets="CopyFilesToOutputDirectory">
-    <ItemGroup>
-      <_testRunnerConfigDestFile Include="$(TargetDir)xunit.console.exe.config" />
-    </ItemGroup>
-
-    <Copy SourceFiles="@(AppConfigWithTargetPath)"
-          DestinationFiles="@(_testRunnerConfigDestFile)"
-          SkipUnchangedFiles="true" />
-  </Target>
-
-  <!-- ResolveAssemblyReferences is the target that populates ReferenceCopyLocalPaths which is what is copied to output directory. -->
-  <Target Name="CopyRunnerToOutputDirectory" BeforeTargets="ResolveAssemblyReferences">
-    <ItemGroup>
-      <!-- Copy test runner to output directory -->
-      <None Include="$([System.IO.Path]::GetDirectoryName('$(XunitConsole472Path)'))\*"
-            Exclude="$([System.IO.Path]::GetDirectoryName('$(XunitConsole472Path)'))\xunit.console.*exe.config;
-                     $([System.IO.Path]::GetDirectoryName('$(XunitConsole472Path)'))\xunit.console.x86.exe;
-                     $([System.IO.Path]::GetDirectoryName('$(XunitConsole472Path)'))\xunit.abstractions.dll"
-            Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(XunitConsole472Path)' != ''"
-            CopyToOutputDirectory="PreserveNewest"
-            Visible="false" />
-
-      <_xunitConsoleNetCoreExclude Condition="'$(GenerateDependencyFile)' != 'true' and '$(XunitConsoleNetCore21AppPath)' != ''" Include="$([System.IO.Path]::GetDirectoryName('$(XunitConsoleNetCore21AppPath)'))\xunit.console.deps.json" />
-      <None Include="$([System.IO.Path]::GetDirectoryName('$(XunitConsoleNetCore21AppPath)'))\*"
```

Comment:
Should we update this: https://github.com/dotnet/runtime/pull/39923/files#diff-4ad56dfe7dc627de24789e0d2f879e67R185 to not exclude xunit assemblies? I think mobile test runners depend on them.